superset/views/csv_import.py

```diff
@@ -225,7 +239,7 @@ def _create_database(self, db_name: str, db_flavor=SQLITE) -> models.Database:
         db_flavor -- which database to use postgres or sqlite
 
         Raises:
-            ValueError: If a file with the database name already exists in the folder
+            ConversionException: If a file with the database name already exists in the folder
```

Comment:
I don't really find that name fitting, why is it a conversion problem when the db-file already exists?